app/src/common/css/variables/screen-size-variable.scss

```diff
@@ -3,7 +3,7 @@ $SCREEN_XS_MAX: 767px;
 $SCREEN_SM_MIN: 768px;
 $SCREEN_SM_MAX: 991px;
 
-//$SCREEN_MD_MIN: 992px;
+$SCREEN_MD_MIN: 992px;
```

Comment:
Do not use ..._MIN variables in your styles . Remove Them from admin/header component too.